### DISCUSSION TITLE
ESE internal COPN/COPX, PEL/XFL

### DISCUSSION THREAD

**Discussion by mjpikat** [pinatacolada/topskylppc#128] [General]
Currently with multiple ACC sectors open, it doesn't coordinate labels correctly, for example with Sector W, E and APP open, traffic with ADEP/ADES LPPT and coming/going to west would coordinate it with sector E, although taking into account climb/descend rate it would rather go to sector W (it also coordinates it with W even when label is already with LPPT APP). It can be done by inputting Sector W into ESE and assigning coordination points (that can be TMA points, for example).

Also, PEL/XFL should be associated for TMA points so that there is no extra coordination on that and label would not produce "XFL/PEL discrepancy" errors.

I would give priority for setting this to LPPT first (COPX), FIR points (as FIR_COPX) and only then setup regional airports COPX-s. These can also be set for departure SID so that by clicking CFL you can pick first published SID altitude (e.g. Porto FL100). Contact me and I will show you how to do it :)

**Comment by mjpikat** [comment]
Not really a TopSky setup issue, but related

**Comment by pinatacolada** [comment]
@becas22 
leaving this as not a bug as its not related so topsky, but we dont really have a git for gng so ¯\_(ツ)_/¯
